src/main/java/uk/gov/hmcts/reform/roleassignment/auditlog/AuditInterceptor.java

```diff
@@ -63,7 +63,9 @@ private AuditContext populateHttpSemantics(AuditContext auditContext,
         context.setHttpStatus(response.getStatus());
         context.setHttpMethod(request.getMethod());
         context.setRequestPath(request.getRequestURI());
-        if ((context.getResponseTime() != null && context.getResponseTime() > 500) || LOG.isDebugEnabled()) {
+        if ((context.getResponseTime() != null && context.getResponseTime() > 500)
+            || (context.getResponseTime() != null && context.getHttpStatus() == 422)
+            || LOG.isDebugEnabled()) {
```

Comment:
why this context.getResponseTime() != null is required on line num 67 where its a combination of context.getHttpStatus() == 422?